sys/shell/commands/sc_gnrc_rpl.c

```diff
@@ -298,6 +299,8 @@ int _gnrc_rpl_dodag_show(void)
                 gnrc_rpl_instances[i].mop, gnrc_rpl_instances[i].of->ocp,
                 gnrc_rpl_instances[i].min_hop_rank_inc, gnrc_rpl_instances[i].max_rank_inc);
 
+#if !IS_USED(MODULE_XTIMER)
```

Comment:
Hmm, if there is no *better* way to access the current time-to-fire of the timer, then we should probably remove the TC print in both cases.  It was there for debugging purposes and is not very important to have that around.

Reply:
No, to my knowledge there is not really a decent way with ztimer to get the TC. Removing all-together sounds good to me, will adapt.